requirements-test.txt

```diff
@@ -1,22 +1,38 @@
 certifi==2016.8.31 \
-#    --hash=sha256:f7708a42d86f29ccc7c8c4ff9d34a8d854d8d78eb2973d1f28406bb43d6b8919
```

Comment:
Ack, I didn't realize things were in such a sorry state here. There's test failures with these changes, and it's really not your fault. Let's just revert this back to the old state, and I'll deal with fixing hashes somewhere else. I think I can push to your branch to fix this and save you the trouble...